### ISSUE TITLE
nonlinear utility function

### ISSUE THREAD

**Issue by duxdfx** [joemolloy/fast-mixed-mnl#11]
I am trying to estimate a mixed logit model with a non-linear utility function (with risk preference parameters to identify). 
Does mixl allow me to specify a non-linear utility function?

**Comment by dheimgartner** [comment]
Yes, you can specify any utility function you like (also with multiplicative terms).
